packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuTop/Features/template.small.tsx

```diff
@@ -23,32 +15,18 @@ const TransactIcon = styled(NavbarNavItemIcon)`
   margin-right: 4px;
 `
 
-const FeaturesSmall = (
-  props: Props & { showModal: (modal: 'SEND' | 'REQUEST') => void }
-) => {
+const FeaturesSmall = (props: Props & { showModal: (modal: 'SEND' | 'REQUEST') => void }) => {
   const ref = useRef(null)
   const [isMenuOpen, toggleIsMenuOpen] = useState(false)
   useOnClickOutside(ref, () => toggleIsMenuOpen(false))
 
   return (
-    <NavbarNavItemButton
-      data-e2e='featuresSmall'
-      onClick={() => toggleIsMenuOpen(!isMenuOpen)}
-    >
-      <TransactIcon
-        persist
-        name='plus-in-circle-filled'
-        size='18px'
-        color='alwaysWhite'
-      />
+    <NavbarNavItemButton data-e2e='featuresSmall' onClick={() => toggleIsMenuOpen(!isMenuOpen)}>
```

Comment:
might want to use `useCallback()` for the `onClick` prop

Reply:
this file is changed due to format on save